### ISSUE TITLE
Feature request: clickable montage

### ISSUE THREAD

**Issue by chneukirchen** [phillipberndt/pqiv#103]
The montage overview is pretty nice, it would be great if we could just click on the thumbnails instead of using the cursor keys, and perhaps scroll it with the mouse wheel?

**Comment by phillipberndt** [comment]
Good idea, I'll see when I find the time! :)

**Comment by phillipberndt** [comment]
Is https://github.com/phillipberndt/pqiv/tree/montage-mouse what you had in mind?

(In fullscreen, this will only work if you have your mouse cursor enabled, e.g. using `--action="set_cursor_auto_hide(2)"`.)

**Comment by chneukirchen** [comment]
Pretty much how I imagined it. I wonder if double-clicking on the thumbnails is needed, or of a single click should open it already?

**Comment by chneukirchen** [comment]
And perhaps the mouse stuff should be enabled in montage mode then by default.

**Comment by phillipberndt** [comment]
The problem with the mouse is that asking X11 to inform you about motion events is really expensive. While moving the mouse, there's an event every 10ms or so, and the resulting latency becomes noticeable especially if using pqiv over the network. That's why I struggled with this request for so long - I agree that the mouse should be enabled by default, but I also wanted to avoid making pqiv unusable over ssh in the process.

I think that I finally found a solution though. I now disable mouse reporting once the user moves the mouse, and then start a timer that queries the mouse position a second later. If the mouse didn't move, then the cursor can be hidden again. This mode is now enabled by default, except in normal (non-montage) fullscreen mode.

I've decided against single click for now, but I'll likely make this configurable in the long run.
